Sources/ChatUI/ChatInChannel/MessageField/MessageField.swift

```diff
@@ -164,14 +165,13 @@ public struct MessageField: View {
                 // Send Button
                 if showsSendButtonAlways || !text.isEmpty {
                     Button(action: onTapSend) {
-                        Image.send.medium
+                        appearance.images.getSend(colorScheme).medium
                     }
                     .frame(width: 36, height: 36)
                     .tint(appearance.tint)
                     .disabled(text.isEmpty)
                 }
             }
-            .padding(16)
```

Comment:
I've removed this as for my app it meant there was too much padding at the top of the messageField, I think it's correct to remove the padding and have it in the calling view.  If this pull request is accepted then I'll happily update the examples to have this padding